tests/driver_bme280/main.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2016 Kees Bakker, SODAQ
+ *               2018 Freie Universität Berlin
+ * Copyright (C) 2022 Mesh4all mesh4all.org
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup     test_group_driver_bme280
```

Comment:
```suggestion
 * @ingroup     tests_driver_bme280
```

Reply:
This was resolved in a past discussion , a new pr should be open and should contain all these changes